TypeCobol.Transform.Test/TransformTest.cs

```diff
@@ -49,33 +68,31 @@ internal static void Test(string workingDirectory, string arguments)
             while (!outputDir.Exists)
                 outputDir.Refresh();
 
-            StringReader sr = new StringReader(arguments);
-            string line_args = null;
             List<int> returnCodes = new List<int>();
-            while ((line_args = sr.ReadLine()) != null)
+            foreach (var argumentList in arguments)
             {
-                line_args = line_args.Trim();
-                if (line_args.Length > 0)
+                System.Diagnostics.Process process = new System.Diagnostics.Process();
+                System.Diagnostics.ProcessStartInfo startInfo = new System.Diagnostics.ProcessStartInfo();
+                startInfo.WindowStyle = System.Diagnostics.ProcessWindowStyle.Hidden;
+                startInfo.FileName = "dotnet";
+                startInfo.WorkingDirectory = workingDirectory;
+                // Find TypeCobol.Transform.dll location
+                string currentDirectory = Environment.CurrentDirectory;
+                string configuration = Path.GetFileName(currentDirectory);
+                string pathToExe = Path.Combine(currentDirectory, "..", "..", "..");
```

Comment:
Maye we can rename `pathToExe` to `transformPath`?

Reply:
done